python/ray/util/client/server/server.py

```diff
@@ -432,7 +434,8 @@ def _prepare_runtime_env(self, job_runtime_env) -> List[str]:
                 try:
                     working_dir = runtime_env.fetch_package(uri)
                     if working_dir:
-                        os.chdir(working_dir)
+                        os.chdir(str(working_dir))
+                        sys.path.insert(0, str(working_dir))
```

Comment:
Could we insert "." instead? (actually, I thought Python would always look into ".", is this not the case?)